File: day1/sol.py
with open("input.txt", "r") as f:
  i = f.read() 
  sol = sorted(map(lambda x: sum(map(int, x.split("\n"))), i.split("\n\n")[:-1]), reverse=True)
  print(sol[0])
  print(sol[0] + sol[1] + sol[2])
  

File: day1/sol.ts
let i = (await Deno.readTextFile("./input.txt")).split("\n\n");

let sol = i
  .slice(0, i.length - 1)
  .map((x) =>
    x
      .split("\n")
      .map(Number)
      .reduce((a, b) => a + b)
  )
  .sort((a, b) => b - a);

console.log(sol[0]);
console.log(sol[0] + sol[1] + sol[2]);
